apps/ejabberd/src/mod_roster.erl

```diff
@@ -339,6 +344,9 @@ create_sub_el(Items, Version) ->
                      {<<"ver">>, Version}],
             children = Items}].
 
+-spec get_user_roster(Acc :: [roster()],
+                     {LUser :: binary(), LServer :: binary()}) ->
+    [{roster()}].
```

Comment:
How is it possible this is acceptable by dialyzer? Would it be enough to have just `[roster()]` as the result type? Why does it need to be surrounded by  `{}`?